packages/ruleset/src/functions/check-major-version.js

```diff
@@ -14,9 +14,27 @@ module.exports = targetVal => {
   if (oas3) {
     const servers = targetVal['servers'];
     if (servers && Array.isArray(servers)) {
-      const urls = servers.map(o => o['url']);
-      const versions = urls.map(url => getVersion(url));
+      // Gather up all the unique path segments that
```

Comment:
I couldn't figure out how to do the try/catch within the list.map() function, so I made the code a little more explicit about what's going on.